easybuild/easyconfigs/l/line_profiler/line_profiler-4.1.1-GCCcore-12.2.0.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'PythonPackage'
+
+name = 'line_profiler'
+version = '4.1.1'
+
+homepage = 'https://github.com/pyutils/line_profiler'
+description = """line_profiler is a module for doing line-by-line profiling
+of functions. kernprof is a convenient script for running either
+line_profiler or the Python standard library's cProfile or profile modules,
+depending on what is available."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+github_account = 'pyutils'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['71c150bf06dd964fc0e5fbc3a5eb31bbee03c535d31c9548999c3e0302dcbb27']
+
+builddependencies = [
+    ('scikit-build', '0.17.2'),
```

Thanks for your contribution, and welcome to EasyBuild! :-) 

I compared this easyconfig to an older version and noticed that you lowered the toolchain from `foss` to `GCCcore`; I think this makes sense, as all dependencies depend on `GCCcore` as well, and this library does not seem to be performance-critical or something. However, the CI is unhappy now, as it expects to find a build dependency on `binutils` for tools built with `GCCcore`. So, the following should be an easy fix to pass the CI checks:

```suggestion
    ('binutils', '2.39'),
    ('scikit-build', '0.17.2'),
```

---

Thanks for your welcome !

I committed your suggestion, it seemed to be compiling fine, so I'm not sure why I need to add binutils.
Is it to ensure the tests pass or is there something more I am missing ?